examples/api/lib/material/color_scheme/color_scheme.0.dart

```diff
@@ -0,0 +1,310 @@
+        ]),
+        divider,
+        ColorGroup(
+          children: <ColorChip>[
+            ColorChip(label: 'error', color: colorScheme.error, onColor: colorScheme.onError),
+            ColorChip(label: 'onError', color: colorScheme.onError, onColor: colorScheme.error),
+            ColorChip(
+                label: 'errorContainer', color: colorScheme.errorContainer, onColor: colorScheme.onErrorContainer),
+            ColorChip(
+                label: 'onErrorContainer', color: colorScheme.onErrorContainer, onColor: colorScheme.errorContainer),
+          ],
+        ),
+        divider,
+        ColorGroup(
+          children: <ColorChip>[
+            ColorChip(label: 'surfaceDim', color: colorScheme.surfaceDim, onColor: colorScheme.onSurface),
+            ColorChip(label: 'surface', color: colorScheme.surface, onColor: colorScheme.onSurface),
+            ColorChip(label: 'surfaceBright', color: colorScheme.surfaceBright, onColor: colorScheme.onSurface),
+            ColorChip(label: 'surfaceContainerLowest', color: colorScheme.surfaceContainerLowest, onColor: colorScheme.onSurface),
+            ColorChip(label: 'surfaceContainerLow', color: colorScheme.surfaceContainerLow, onColor: colorScheme.onSurface),
+            ColorChip(label: 'surfaceContainer', color: colorScheme.surfaceContainer, onColor: colorScheme.onSurface),
+            ColorChip(label: 'surfaceContainerHigh', color: colorScheme.surfaceContainerHigh, onColor: colorScheme.onSurface),
+            ColorChip(label: 'surfaceContainerHighest', color: colorScheme.surfaceContainerHighest, onColor: colorScheme.onSurface),
+            ColorChip(label: 'onSurface', color: colorScheme.onSurface, onColor: colorScheme.surface),
+            ColorChip(
+                label: 'onSurfaceVariant', color: colorScheme.onSurfaceVariant, onColor: colorScheme.surfaceContainerHighest),
+          ],
+        ),
+        divider,
+        ColorGroup(
+          children: <ColorChip>[
+            ColorChip(label: 'outline', color: colorScheme.outline),
+            ColorChip(label: 'shadow', color: colorScheme.shadow),
+            ColorChip(
+                label: 'inverseSurface', color: colorScheme.inverseSurface, onColor: colorScheme.onInverseSurface),
+            ColorChip(
+                label: 'onInverseSurface', color: colorScheme.onInverseSurface, onColor: colorScheme.inverseSurface),
+            ColorChip(label: 'inversePrimary', color: colorScheme.inversePrimary, onColor: colorScheme.primary),
+          ],
+        ),
+      ],
+    );
+  }
+}
+
+class ColorGroup extends StatelessWidget {
+  const ColorGroup({super.key, required this.children});
+
+  final List<Widget> children;
+
+  @override
+  Widget build(BuildContext context) {
+    return RepaintBoundary(
+      child: Card(clipBehavior: Clip.antiAlias, child: Column(children: children)),
+    );
+  }
+}
+
+class ColorChip extends StatelessWidget {
+  const ColorChip({
```

Comment:
Given the way this is used, a constructor with position arguments would be a little more compact.